src/Jellyfin.Database/Jellyfin.Database.Providers.Sqlite/Migrations/JellyfinDbModelSnapshot.cs

```diff
@@ -1388,7 +1388,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<double?>("Rating")
                         .HasColumnType("REAL");
 
-                    b.Property<DateTimeOffset?>("RetentionDate")
+                    b.Property<DateTime?>("RetentionDate")
```

Comment:
we should not do stuff like this when we support providers that actually understand the difference but for sqlite its _fine_